src/components/03-components/04-labels/labels--outline.njk

```diff
@@ -1,37 +1,81 @@
-  </div>
-  <div class="grid-col-auto">
-    <h1 class="sds-type--label-title">Status</h1>
-    <span class="sds-tag sds-tag--outline-blue">Submitted</span>
-  </div>
-  <div class="grid-col-auto">
-    <h1 class="sds-type--label-title">Status</h1>
-    <span class="sds-tag sds-tag--outline-yellow">In Progress</span>
+<h3>Outline Labels</h3>
+<div class="grid-row flex-align-end">
+  <div class="grid-col-auto margin-top-2">
+    <span class="sds-type--label-title">Expiration Date</span>
+    <span class="sds-tag sds-tag--alt">Feb 20, 2019</span>
   </div>
-  <div class="grid-col-auto">
-    <h1 class="sds-type--label-title">Status</h1>
+  <div class="grid-col-auto margin-top-2">
+    <span class="sds-type--label-title">Status</span>
     <span class="sds-tag sds-tag--outline-light-blue">Draft</span>
   </div>
-  <div class="grid-col-auto">
-    <h1 class="sds-type--label-title">Status</h1>
-    <span class="sds-tag sds-tag--outline-red">Expired</span>
-  </div>
 </div>
+</p>
 
-<h3>Outline Label Options</h3>
+<h3>Status Outline Labels</h3>
+<div class="grid-row">
+<div class="grid-col-auto margin-top-2">
+  <span class="sds-type--label-title">Status</span>
+  <span class="sds-tag sds-tag--outline-green">Active</span>
+</div>
+<div class="grid-col-auto margin-top-2">
+  <span class="sds-type--label-title">Status</span>
+  <span class="sds-tag sds-tag--outline-light-blue">Draft</span>
+</div>
+<div class="grid-col-auto margin-top-2">
+  <span class="sds-type--label-title">Status</span>
+  <span class="sds-tag sds-tag--outline-blue">Submitted</span>
+</div>
+<div class="grid-col-auto margin-top-2">
+  <span class="sds-type--label-title">Status</span>
+  <span class="sds-tag sds-tag--outline-yellow">In Progress</span>
+</div>
+<div class="grid-col-auto margin-top-2">
+  <span class="sds-type--label-title">Status</span>
+  <span class="sds-tag sds-tag--outline-red">Expired</span>
+</div>
+<div class="grid-col-auto margin-top-2">
+  <span class="sds-type--label-title">Status</span>
+  <span class="sds-tag sds-tag--outline-brown">Deactivated</span>
+</div>
+<div class="grid-col-auto margin-top-2">
+  <span class="sds-type--label-title">Status</span>
+  <span class="sds-tag sds-tag--outline-dark-gray">Cancelled</span>
+</div>
+</div>
+<h3>Status Outline Labels (Long Name)</h3>
 <div class="grid-row flex-align-end">
-  <div class="grid-col-auto">
-    <h1 class="sds-type--label-title">Expiration Date</h1>
-    <span class="sds-tag sds-tag--alt">Feb 20, 2019</span>
+<div class="grid-col-auto margin-top-2">
+  <span class="sds-type--label-title">Status</span>
+  <span class="sds-tag sds-tag--outline-yellow">Work In Progress</span>
+</div>
+</div>
+
+<h3>Alert Outline Label Options</h3>
+<div class="grid-row flex-align-end">
+  <div class="grid-col-auto margin-top-2">
+    <span class="sds-type--label-title">Expiration Date</span>
+    <span class="sds-tag sds-tag--alt sds-tag--border-blue">Feb 20, 2019</span>
```

Comment:
can we please change all instances of alt to alert, alt makes me think alternative